### PR TITLE
Change units in SA time profiles

### LINKED ISSUE #610: SA: Time profiles - change units
Best support can be achieved when using the plotIndividualTimeProfile() from ospsuite-r

### PR DESCRIPTION
**Pull request by rengelke**
Feature to change units in sensitivityTimeProfiles()
Closes #610 

@param xUnits, yUnits  Lists of units for the x-axis and y-axis, respectively. If a list of length one is provided, it will be applied to all `outputPaths` if conversion is possible. If a list of multiple units is provided, the units list should correspond to the `outputPaths`, and units conversion will be applied accordingly. If `NULL`, default units from the simulation results will be used.

### CODE REVIEW

**Review by Felixmil — COMMENTED** [review]
Do you thinkg it possible to add tests to improve coverage on the new code ? There are 3 warnings from codecov.

**Comment by rengelke** [comment]
> Do you thinkg it possible to add tests to improve coverage on the new code ? There are 3 warnings from codecov.

Added tests; 1 codecov warning should be gone. The remaining 2 will be fixed in a separate PR.

File: R/sensivitity-time-profiles.R [hunk 1]
```diff
@@ -499,3 +515,82 @@ sensitivityTimeProfiles <- function(sensitivityCalculation,
...
+  unitsConvertable <- TRUE
+
+  tryCatch(
+    {
+      convertUnits(dataCombined, xUnit = xUnit, yUnit = yUnit)
+    },
+    error = function(e) {
+      unitsConvertable <<- FALSE
+    }
+  )
```
Comment (Felixmil) [inline R/sensivitity-time-profiles.R:539]: Another approach, FYI:

```r
ans <- try(
  {
    convertUnits(dataCombined, xUnit = xUnit, yUnit = yUnit)
  },
  silent = TRUE
)

if (inherits(ans, "try-error")) unitsConvertable <- FALSE
```

**Review by Felixmil — APPROVED** [review]

File: R/sensivitity-time-profiles.R (hunk 1 above)
Comment (rengelke) [reply R/sensivitity-time-profiles.R:539]: Yes, it's an alternative

**Pull request merged**